notebooks/00_quick_start/xdeepfm_movielens.ipynb

```diff
@@ -0,0 +1,1541 @@
+{
```

Comment:
<p>this might break if the folders are not created, probably it is safer to do just `data_path = "."`</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/861/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='17'/>